### ISSUE TITLE
Education rewriting/redirects - legacy

### ISSUE THREAD

**Issue by AnneHurley** [department-of-veterans-affairs/va.gov-team#7327]
Copied from old vets.gov ticket #17828

**User story**
As a Veteran considering or actively using the GI Bill and other VA education benefits, I want to quickly and easily see exactly how much tuition my benefits cover for different types of education and training, and how much money I can get for additional needs like housing and books.
Steps to completion
 
 

- Strategy recommendations @DanielleSoCompany
-  
-  Write page @AnneHurley (w/ support from Danielle as needed)
-  
-  Copy edit and build page in Github (@RLHecht with support from @bethpotts)
-  
-  Strategy/voice edit @DanielleSoCompany
-  
-  Final proof
-  
-  Send to SMEs for review
-  
-  SME feedback received
-  
-  Implement SME feedback
-  
-  Final internal review and publish

**IA details**
Where this section will live: This section will live in the "More Information and Resources" section. The existing Post-9/11 GI Bill and Other VA Education Benefit Rates page will be the main page in this section. The specific benefit pages will live beneath this page.

**Left nav labels/IA structure :**
Benefit Rates 
Post-9/11 GI Bill and Fry Scholarship
Montgomery GI Bill - Active Duty
Montgomery GI Bill - Selected Reserve
$600 Buy-up program
Reserve Educational Assistance Program
Survivors' and Dependents' Educational Assistance Program
GI Bill and Other Education Housing Benefits
Residential requirements [https://www.benefits.va.gov/gibill/post911_residentraterequirements.asp]
We'll also crosslink to/from this section from other pages within Education and Training.

**Content details**
Section main page: Post-9/11 GI Bill and Other VA Education Benefit Rates
Source page: https://www.va.gov/education/benefit-rates/
SEO keywords: post 911 gi bill rates, va education benefits rates, va benefit rates

**Directions:**
Add major links/teasers for each of the benefit rates sub pages.
Reframe the GI Bill Comparison link as a related link.
Add a related link to the new Compare Education Benefits page).
Revise intro as needed to accommodate page changes noted above.
1st benefit rate subpage: Post-9/11 GI Bill (Chapter 33) and Fry Scholarship Rates
For academic year 2019 (August 1, 2019 to July 31, 2020)
Source page: https://www.benefits.va.gov/GIBILL/resources/benefits_resources/rates/ch33/ch33rates080119.asp
SEO keywords: post 911 gi bill rates, gi bill housing allowance, post 911 gi bill pay chart, gi bill bah rates, post 911 gi bill bah rates, gi bill pay rates, gi bill payment rates

- What's needed for rewriting:
- Incorporate the information and rate tables from: https://www.benefits.va.gov/GIBILL/resources/benefits_resources/rates/ch33/ch33rates080119.asp. Edit to align with plain language and VA.gov style standards.
- Optimize for SEO.
- Make clear that these rates also apply to the Fry Scholarship.
- Consider how we might structure the page so that the user can easily get to/see the info they need without feeling overwhelmed, but also avoiding hiding pertinent info.
- Consider how we might include the info on prorated rates and the chart from https://gibill.custhelp.va.gov/app/answers/detail/a_id/947. Could potentially be a question ("Will I get the full rate for these benefits?") or an accordion under each table.

Note that the “GI Bill resident rate and requirements” page (https://www.benefits.va.gov/gibill/post911_residentraterequirements.asp) is a page that we will be talking with SMEs about. For now, please incorporate just a brief note with the key info that a Veteran would need to know, and don't link to this page for more info.
The page links out to https://gibill.custhelp.va.gov/app/answers/detail/a_id/1412/ and https://gibill.custhelp.va.gov/app/answers/detail/a_id/1480 for more info on housing allowance rates. Explore whether we can incorporate that additional info into this page while keeping content brief and focused.

Include links to three historic benefit rate tables, at the bottom of the page. We will be asking SMEs if these can be made into PDFs, but for now we'll link to legacy pages.

Crosslink to other pages (Yellow Ribbon, pages within How to Use Your Benefits as relevant.
2nd benefit subpage: Montgomery GI Bill - Active Duty (Chapter 30) rates
**Effective October 1, 2018** **NOTE: If rates are still from 2018 they will all need to be updated**

Source page: https://www.benefits.va.gov/GIBILL/resources/benefits_resources/rates/ch30/ch30rates100118.asp
SEO keywords: montgomery gi bill rates, mgib rates, chapter 30 gi bill rates
What's needed for rewriting:
Same as above for all general direction re: tables, plain language, historic rate tables, cross-linking, etc.
Work in crosslink to new $600 buy-up program rates page.

3rd benefit subpage: Montgomery GI Bill - Selected Reserve (Chapter 1606) rates
Effective October 1, 2018
Source page: https://www.benefits.va.gov/GIBILL/resources/benefits_resources/rates/ch1606/ch1606rates100118.asp
SEO keywords: reserve gi bill rates, chapter 1606 rates, chapter 1606 gi bill rates, mgib sr rates, montgomery gi bill selected reserve rates
What's needed for rewriting:
Same as above for all general direction re: tables, plain language, historic rate tables, cross-linking, etc.

4th benefit subpage: $600 Buy-up program rates
Source page: https://www.benefits.va.gov/gibill/resources/benefits_resources/rates/600_buyup.asp
SEO keywords: gi bill buy up program
What's needed for rewriting:
Same as above for all general direction re: tables, plain language, historic rate tables, cross-linking, etc.

5th benefit subpage: Survivors' and Dependents' Educational Assistance Program (Chapter 35) rates
Source page: https://www.benefits.va.gov/GIBILL/resources/benefits_resources/rates/ch35/ch35rates100118.asp
SEO keywords: chapter 35, chapter 35 rates, chapter 35 benefits rates, dependents educational assistance rates, chapter 35 pay rates

What's needed for rewriting:
Same as above for all general direction re: tables, plain language, historic rate tables, cross-linking, etc.
Note: Will confirm approach to REAP rates page shortly.
Update: Confirmed. We'll add REAP page for now.

6th benefit subpage: Reserve Educational Assistance Program rates
Source page: https://www.benefits.va.gov/GIBILL/resources/benefits_resources/rates/ch1607/ch1607rates100118.asp
SEO keywords: chapter 1607 rates, reap
What's needed for rewriting:
Same as above for all general direction re: tables, plain language, historic rate tables, cross-linking, etc.

7th benefit subpage: GI Bill and other education housing benefit rates
Source page: TBD - Anne to provide
SEO keywords: gi bill housing allowance, gi bill bah rates, post 911 gi bill bah rates, gi bill pay rates, gi bill payment rates
What's needed for rewriting:
Same as above for all general direction re: tables, plain language, cross-linking, etc.
Github details
Branch: https://github.com/department-of-veterans-affairs/vagov-content/tree/content-WIP-rewriting-education
PR: department-of-veterans-affairs/vagov-content#375
Heroku: https://vagov-content-pr-375.herokuapp.com/

**Comment by AnneHurley** [comment]
Screen shot of 3 pages already built in Drupal but not live: 
<img width="1221" alt="Screen Shot 2020-03-27 at 2 56 40 PM" src="https://user-images.githubusercontent.com/46570591/77790631-642b9c00-7022-11ea-9291-8da92095978b.png">
2019 Post-9/11 GI Bill (Chapter 33) rates
2019 REAP (Chapter 1607) rates
2019 VA education benefit rates

^ Will need to be updated with2020 rates

**Comment by AnneHurley** [comment]
See archived issue https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17828 and comment thread. It appears we have not regrouped with @jenniferlee-dsva after SME review. Some pages were built in GH and the three above have been built in Drupal.

**Comment by AnneHurley** [comment]
Rewritten education pages built in GitHub and not yet migrated to Drupal mentioned in this PR: https://github.com/department-of-veterans-affairs/vagov-content/pull/375

**Comment by mnorthuis** [comment]
@AnneHurley Can we separate out the writing work from the redirect please?  I don't believe I need to be on this issue.

**Comment by AnneHurley** [comment]
@mikki This ticket duplicates the expired ticket https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17828   - And that ticket is made up of all the education URLs needing to be redirected in our current Education Redirect ticket, which you are on. That's why I added you. Feel free to take yourself off.

**Comment by DanielleThierryUSDSVA** [comment]
Hi @AnneHurley @RLHecht. I'd like to regroup on these education rates tables as I pick back up on the rewriting initiative. Once we've answered these questions, I can create a new issue detailing the specific steps we'll need to take to complete the rewriting of this section.

How's Tuesday 4/28 look for you both to meet and talk through this section? I could do 11:00 a.m. to 12:00 noon ET if that works for you.

From my understanding, these are the initial questions we'll need to go over. Please feel free to add others.

1) There was a note in the old ticket about being blocked because the pages were in GitHub and not Drupal. It didn't look like we'd decided how to go about getting them in Drupal. So we need to know if we should rebuild the pages in Drupal, or if they can be migrated over, correct? Could you talk with Beth and Howie to get their recommendations?

2) Another issue was that we wanted to find an alternative to building/updating tables manually. We now have that, so we can move forward with these now, correct? Anything specific about the rate tables on these particular pages we should consider in terms of the updating solution?

3) I also wonder if there may be anything we need to rethink to make these pages work within the confines of Drupal components. Can you please look over the pages as they currently are in GitHub and flag anything that looks like it may be problematic from a Drupal component standpoint?

Thank you!

FYSA @stephorkandatx @jenniferlee-dsva

**Comment by RLHecht** [comment]
@DanielleThierryUSDSVA That time tomorrow works for me. I asked Nick to help me get the Heroku working for these pages so we can take a look and see if there are any anomalies that will cause issues with building in Drupal. I can take a look before tomorrow so we can go over it.

**Comment by DanielleThierryUSDSVA** [comment]
Thank you so much, @RLHecht!

**Comment by AnneHurley** [comment]
@DanielleThierryUSDSVA  That time works for me also. @RLHecht  Since I really never built much in GH/Heroku, I'd like to refresh my memory. Maybe after you meet with Nick re the above? Or if I could sit in on that meeting too that might help me get my arms around it - if that is convenient for you.

**Comment by DanielleThierryUSDSVA** [comment]
Excellent, thank you, @AnneHurley! I just sent you both an invite for 11 - 11:30 tomorrow...I just got added to a covid meeting from 11:30 - 12, but we can find another 30 minutes later in the day if we need more time to talk through everything. Thanks!

**Comment by RLHecht** [comment]
@ncksllvn or @kelsonic would you guys be able to get the Heroku up and running for this PR like you did for me with Pension last week? This is it:  https://github.com/department-of-veterans-affairs/vagov-content/pull/375

**Comment by ncksllvn** [comment]
@RLHecht Done! But LMK if something looks off as a result.

**Comment by DanielleThierryUSDSVA** [comment]
**Notes from 4.28.20 meeting with Randi, Anne, and Danielle:**

1. Anne and Randi will rebuild pages in Drupal, and will sync with Beth for tips, best practices, etc.
2. We'll move ahead with building out these pages in parallel with Anne's work on CSV files for rates tables. Anne will work that into the process for this section as we go.
3. Anne and Randi will flag content anomalies and we'll address them as needed as we build out these pages in Drupal.
4. We'll need to reflect the 2020 rates in these pages.
5. Danielle will create new issues for each page in the Education rewriting epic so we can easily track our steps.

Thanks, @AnneHurley and @RLHecht! And thanks, @ncksllvn, for the heroku link!

FYSA @stephorkandatx
